From 20ee1f6fd5b4117cbb17d54790e78a1cb79ec342 Mon Sep 17 00:00:00 2001
From: Gabriel de Perthuis <g2p.code@gmail.com>
Date: Fri, 27 Oct 2023 14:15:56 +0200
Subject: [PATCH 08/12] Fix error compatibility alias (#179)

In #167, the error type was renamed from RcgenError to rcgen::Error.

Make sure the compatibility alias uses the correct name.
---
 rcgen/CHANGELOG.md | 2 +-
 rcgen/src/lib.rs   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/rcgen/CHANGELOG.md
+++ b/rcgen/CHANGELOG.md
@@ -3,7 +3,7 @@
 
 ## Unreleased
 
-- Rename `RcGenError` to `Error` to avoid stuttering when used fully-qualified via `rcgen::`.
+- Rename `RcgenError` to `Error` to avoid stuttering when used fully-qualified via `rcgen::`.
 - Upgrade to `ring` `v0.17`.
 
 ## Release 0.11.3 - October 1, 2023
--- a/rcgen/src/lib.rs
+++ b/rcgen/src/lib.rs
@@ -67,7 +67,7 @@ pub use crate::sign_algo::SignatureAlgor
 #[deprecated(
 	note = "Renamed to `Error`. We recommend to refer to it by fully-qualifying the crate: `rcgen::Error`."
 )]
-pub type RcGenError = Error;
+pub type RcgenError = Error;
 
 /// A self signed certificate together with signing keys
 pub struct Certificate {
